frontend/test/__support__/e2e/cypress_data.js

```diff
@@ -172,3 +172,24 @@ export const QA_DB_CONFIG = {
     },
   },
 };
+
+export const TESTING_DB_CONFIG = {
```

Comment:
Nit: WDYT about calling this `WRITABLE_DB_CONFIG`, that is calling that database "writable".

I had to double-check a few times in my head which one is QA and which one is testing DB.